SpatialGDK/Source/SpatialGDK/Private/Interop/SpatialPlayerSpawner.cpp

```diff
@@ -27,12 +29,24 @@ void USpatialPlayerSpawner::Init(USpatialNetDriver* InNetDriver, FTimerManager*
 	NumberOfAttempts = 0;
 }
 
-void USpatialPlayerSpawner::ReceivePlayerSpawnRequest(FString URLString, const char* CallerAttribute, Worker_RequestId RequestId )
+void USpatialPlayerSpawner::ReceivePlayerSpawnRequest(Schema_Object* Payload, const char* CallerAttribute, Worker_RequestId RequestId )
 {
+	// Extract spawn parameters.
+	FString URLString = GetStringFromSchema(Payload, 1);
```

Comment:
Hmm, I preferred this function not having deserialization code. Could we have a type with the 3 parameters which are passed in?